### DISCUSSION TITLE
12.4. Huoltoikkuna

### DISCUSSION THREAD

**Discussion by CodoDerDritte** [KohaSuomi/Koha#491] [Huolto- ja häiriötiedotteet]
Keskiviikon 12.4. huollossa yhtenäistetään Koha-installaatioiden tietokantaskeemat. Tämä mahdollistaa jatkossa joustavammat tiistaipäivitykset ja vähentää virheiden mahdollisuuksia päivityksissä. Myös tietokantojen ylläpito helpottuu, kun kaikki tietokannat ovat rakenteellisesti ja tietotyypeiltään samanlaisia keskenään. Muutosten kohteena olevat tietokannan taulut lukitaan muutosten ajaksi, joka voi näkyä muutamien minuuttien viiveinä Kohan toiminnassa. Katkoihin kannattaa siis varautua.

**Comment by CodoDerDritte** [comment]
Muistattehan lisäksi, että ajastettuja tehtäviä ei ajeta huoltokatkojen aikana. Esimerkiksi hyllyvarausraportti päivittyy huoltopäivinä vasta klo 9.10 alkaen.

**Comment by CodoDerDritte** [comment]
Huoltoikkunan aikana tallennusjärjestelmän vikaantunut ohjausyksikkö vaihdetaan uuteen. Ohjausyksiköt on kahdennettu, joten vikaantumisella ei ole ollut vaikutusta järjestelmän käyttöön. Kaiken mennessä suunnitellusti myöskään vikaantuneen yksikön vaihdolla ei pitäisi olla havaittavaa vaikutusta järjestelmän käyttöön.

Kysymyksessä on takuuvaihto ja toimenpiteen suorittaa palvelinvalmistajan asentaja yhteistyössä tallennusjärjestelmäasiantuntijan kanssa. Koha-Suomella/konesalioperaattorilla ei ole roolia vaihdossa. Vaihto suoritetaan klo 8.30 alkaen.

**Comment by emta001** [comment]
Tietokantoihin on nyt saatu tehtyä suurin osa suunnitelluista ajoista. Outista jäi vielä muutama muutos tekemättä, mutta ne eivät ole mitenkään kriittisiä ja voidaan ajaa myöhemmin jonain hiljaisena hetkenä.  Pidemmät ajot tehtiinkin jo eilen illalla (11.4.), joten jos tulee palautetta Kohan hidasteluista klo 22 jälkeen, niin se johtuu oletettavasti noista ajoista.

**Comment by CodoDerDritte** [comment]
Tallennusjärjestelmän ohjausyksikkö on nyt myös vaihdettu. Huolto tämän kuun osalta valmis.
